services/db/recovery.go

```diff
@@ -149,6 +151,23 @@ func alertOps(mode string) {
 	}
 }
 
+// sendRecoveryModeStat sends the recovery mode metric every 10 seconds
+func sendRecoveryModeStat() {
+	recoveryModeStat := stats.NewStat("recovery.mode_normal", stats.GaugeType)
+	for {
+		time.Sleep(10 * time.Second)
+		switch CurrentMode {
+		case normalMode:
+			recoveryModeStat.Gauge(1)
+		case degradedMode:
+			recoveryModeStat.Gauge(2)
+		case maintenanceMode:
+			recoveryModeStat.Gauge(3)
```

Comment:
let's add a default unknown mode stat?

Reply:
is it possible? I don't know the logic.

Reply:
Don't think we need unknown mode. Recovery mode can only be one of those three modes now. In case it is not one of those three, we will get an alert anyway because of missing data for the metric, right?

Reply:
Yeah you are right. We need to put alert if we don't get metric. Looks good to me. 

Approved